Example/Pods/UberSignals/UberSignals/UBSignal.h

```diff
@@ -195,6 +195,7 @@ CreateSignalInterface(UBMutableDictionarySignal, NSMutableDictionary *mutableDic
 - (instancetype)initWithProtocol:(Protocol *)protocol NS_DESIGNATED_INITIALIZER;
 
 - (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
```

Comment:
Sure, I'll removed the change from my commit, will raise a separate on for UBSignal.h later on. Thanks!